code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -310,6 +310,7 @@
 	back = /obj/item/storage/backpack
 	ears = /obj/item/device/radio/headset/syndicate/alt
 	l_pocket = /obj/item/pinpointer/nuke/syndicate
+	r_pocket = /obj/item/kitchen/knife/combat
```

Comment:
put it in their bag instead, that way they still get it with the full kit outfit.